agent/util-scripts/samples/pbench-copy-results/test-49/opt/pbench-agent/id_rsa

```diff
@@ -0,0 +1 @@
+foo
```

Comment:
Is there a reason why we need this `id_rsa` file for this test?  Seems like this would be required by `ssh` but we really don't need it since we don't invoke `ssh`.  At least, that it what one would think for unit tests.

Reply:
pbench-copy-results checks and fails if it does not exist.